app/filters/strings.js

```diff
@@ -100,6 +100,30 @@ filters.startsWith = (string, target) => {
   }
 }
 
+// Make a string possessive
+// {{ "James Joyce" | possessive }}
+//     James Joyce’s
+// {{ "Joyce James" | possessive }}
+//     Joyce James’
+// {{ "JAMES JOYCE" | possessive }}
+//     JAMES JOYCE’S
+// {{ "Joyce James" | possessive }}
+//     JOYCE JAMES’
+
+filters.possessive = (string) => {
+  const lastLetterOfString = string.split('').slice(-1).toString()
```

Comment:
```suggestion
  const lastLetterOfString = string.split('').slice(-1).toString().toLowerCase()
```

Just convert this to lowercase and then later compare to lowercase `s`